CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,24 @@
 - Update COAP librery to version 0.21.0
 - Using precise dependencies (~=) in packages.json
 - Update MongoDB to 2.2.35
+- Fix: updating dependencies due to known vulnerabilities in the previous ones
+	async: 1.5.2 -> 2.6.1
+    underscore: 1.7.0 -> 1.9.1
+    logops: 0.5.0 -> 1.0.0
+    coap: 0.5.0 -> 1.0.0
+    coap-cli: 0.5.0 -> 1.0.0
+    mongodb: 2.2.35 -> 3.1.8
```

Comment:
mongodb and mongoose dependencies has changed in this PR. Has the comment at https://github.com/telefonicaid/iotagent-node-lib/pull/684#issuecomment-435838531 been taken into account? Have you tested with a MongoDB replica set as DB to see if it works? (I'm afraid that Travis is not so flexible to do that... but in a local environment it should be possible).

Reply:
Fixed with b3fd0fd744c4c6c0131fb4a33118744aaa978a4d

Reply:
I have just tested this with https://github.com/telefonicaid/lightweightm2m-iotagent/pull/141 agains a replica set in my local development environment and it works as you can see in the attached screenshot.

https://github.com/telefonicaid/lwm2m-node-lib/commit/b3fd0fd744c4c6c0131fb4a33118744aaa978a4d just implements the same changes of https://github.com/telefonicaid/iotagent-node-lib/pull/695

![lwm2m_replica_mongo](https://user-images.githubusercontent.com/11388886/48252950-6168e080-e406-11e8-9967-f90e37977f51.png)

Reply:
Looks nice. In addition, I see the changes in lib/services/model/dbConn.js are the same at https://github.com/telefonicaid/iotagent-node-lib/pull/695.

NTC